docs/get-started/using-with-map.md

```diff
@@ -35,10 +40,16 @@ If you are using react-map-gl, there are several ways to provide a token to your
 * Pass it as a prop to the ReactMapGL instance `<ReactMapGL mapboxApiAccessToken={TOKEN} />`
 
 
-### Display Maps Without A Mapbox Token
+### Using other basemap services, or your own
 
 It is possible to use the map component without the Mapbox service, if you use another tile source (for example, if you host your own map tiles). You will need a custom Mapbox GL style that points to your own [vector tile source](https://www.mapbox.com/mapbox-gl-js/style-spec/), and pass it to `ReactMapGL` using the `mapStyle` prop. This custom style must match the schema of your tile source. 
 
+- [CARTO free basemaps](https://carto.com/basemaps) for non commercial applications. There are 3 basemaps available [Positron, Voyager and Dark Matter](/docs/api-reference/carto/basemap)
```

Comment:
I would avoid mentioning the number or names of map styles. Styles may get added/removed later and we won't remember to update this line.

Reply:
Internal links must use the full filename (basemap.md)